cabal.project

```diff
@@ -291,44 +135,48 @@ source-repository-package
     io-sim
     strict-stm
 
+-- Temporary patch for version 0.1.0.0 (181601bc3d9e9d21a671ce01e0b481348b3ca104)
+-- in order to change io-sim from 606de33fa2f467d108fb1efb86daeb3348bf34e3
+-- to 57e888b1894829056cb00b7b5785fdf6a74c3271 to include the ReaderT fix.
+--
+-- TODO [ADP-2447] This source-repository-package should be removable with next node bump.
 source-repository-package
   type: git
-  location: https://github.com/input-output-hk/typed-protocols
-  tag: 181601bc3d9e9d21a671ce01e0b481348b3ca104
-  --sha256: 1lr97b2z7l0rpsmmz92rsv27qzd5vavz10cf7n25svya4kkiysp5
+  location: https://github.com/Anviking/typed-protocols.git
```

Comment:
Hi @Anviking!

Just curious as to the reason for depending on this fork of `typed-protocols` here?
GitHub claims that:
```
This branch is up to date with input-output-hk/typed-protocols:main. 
```
Were you planning to add more features to it? 😃

Reply:
It's explained just above. It's for updating a dependency (`io-sim`), and should no longer be needed with the next node bump.

I had preferred using a branch in the iohk repo, but I don't have push access, and as it will go away next bump, I think this is fine.

Reply:
I was also surprised that a `source-repository-package` for `io-sim` wasn't enough. _Double-checking this fork can't be dropped..._